bsv/PcieEndpointS5.bsv

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2014 xxx
```

Comment:
I'm falling down on the reviewer job. Please add the MIT License text to this file. Thank you for updating the copyright notice.